subsys/net/ip/Kconfig

```
@@ -426,6 +426,14 @@ config NET_MAX_MCAST_ROUTES
 	  This determines how many entries can be stored in multicast
 	  routing table.
 
+config NET_MCAST_ROUTE_MLD_REPORTS
+	bool "Report multicast routes as a part of MLDv2 reports"
+	depends on ((NET_ROUTE_MCAST) && (NET_IPV6_MLD))
+	default n
```

Omit `default n`, and preferably split dependency lines.

```suggestion
	depends on NET_ROUTE_MCAST
	depends on NET_IPV6_MLD
```